src/view/application/Application.js

```diff
@@ -0,0 +1,5 @@
+const Application = () => {
```

Comment:
You are in the `application` folder, rename this file in `index.jsx`

Reply:
done